helix-rest/src/main/java/org/apache/helix/rest/server/resources/helix/ClusterAccessor.java

```diff
@@ -240,7 +240,15 @@ public Response updateCluster(@PathParam("clusterId") String clusterId,
       helixAdmin.manuallyEnableMaintenanceMode(clusterId, command == Command.enableMaintenanceMode,
           content, customFieldsMap);
       break;
-
+    case enableWagedRebalanceForAllResources:
```

Comment:
Question, we will require users to enable maintenance mode before calling this cmd, right?
Shall we comment here or just in the doc? Either one works for me.

Reply:
Why would it be required? 

That sounds more appropriate for the user manual though.

Reply:
This is for reducing the potential partition movement. If the operations are not done in the maintenance mode, the partitions could be moved several times during the rebalance.